cmd/app/main.go

```diff
@@ -44,7 +47,35 @@ func main() {
 		IdleTimeout:  appConf.Server.TimeoutIdle,
 	}
 
-	if err := s.ListenAndServe(); err != nil && err != http.ErrServerClosed {
-		logger.Fatal().Err(err).Msg("Server startup failed")
+	serverErrors := make(chan error, 1)
+
+	go func() {
```

Comment:
@josemkorivo Hi, 
Btw why you need to run `ListenAndServe()` in a separate goroutine? From where you see this pattern? 
As I know, it's a blocking and has bundled with the port we given. I think there is no special benefit on this.